Resources/Textures/LobbyScreens/attributions.yml

```diff
@@ -57,3 +57,8 @@
   license: "Custom"
   copyright: "p.r.ospector on discord / Prospector @ DeltaV"
   source: "https://discord.gg/deltav"
+
+- files: ["deltavf2.png"]
+  lisence: "custom"
```

Comment:
```suggestion
  license: "custom"
```

Reply:
Thank you for telling me, do I edit the PR?

Reply:
you can just push changes to the branch you dont have to close it

Reply:
I didn't know how to do that so I made a new PR.